molecule/testinfra/common/test_fpf_apt_repo.py

```diff
@@ -45,11 +45,7 @@ def test_fpf_apt_repo_fingerprint(host):
 
     c = host.run('apt-key finger')
 
-    fpf_gpg_pub_key_info = """/etc/apt/trusted.gpg.d/securedrop-keyring.gpg
----------------------------------------------
-pub   4096R/00F4AD77 2016-10-20 [expires: 2021-06-30]
-      Key fingerprint = 2224 5C81 E3BA EB41 38B3  6061 310F 5612 00F4 AD77
-uid                  SecureDrop Release Signing Key"""
+    fpf_gpg_pub_key_info = "2224 5C81 E3BA EB41 38B3  6061 310F 5612 00F4 AD77"
```

Comment:
Having the full output seems more secure, especially if the testinfra tests are being used to verify a prod system.

Reply:
The output is different in both `Xenial` and `Focal`, that is why I took this path.